src/server/services/buurt/datasets.ts

```diff
@@ -406,8 +407,43 @@ export const datasetEndpoints: Record<
       cancelTimeout: 30000,
     },
   },
+  laadpalen: {
+    listUrl:
+      'https://map.data.amsterdam.nl/maps/oplaadpunten?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=ms:normaal_beschikbaar&OUTPUTFORMAT=geojson&SRSNAME=urn:ogc:def:crs:EPSG::4326',
+    transformList: transformLaadpalenResponse,
+    transformDetail: transformLaadpalenDetailResponse,
+    idKeyList: 'id',
+    featureType: 'Point',
+    cacheTimeMinutes: BUURT_CACHE_TTL_1_WEEK_IN_MINUTES,
+    geometryKey: 'geometry',
+    triesUntilConsiderdStale: DEFAULT_TRIES_UNTIL_CONSIDERED_STALE,
+    additionalStaticFieldNames: ['connector_type', 'charging_cap_max', 'url'],
+    requestConfig: {
+      headers: {},
+      request: fetchLaadpalen,
+      cancelTimeout: 30000,
+      nextUrls: [
+        'https://map.data.amsterdam.nl/maps/oplaadpunten?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=ms:snel_beschikbaar&OUTPUTFORMAT=geojson&SRSNAME=urn:ogc:def:crs:EPSG::4326',
+      ],
+    },
```

Comment:
```suggestion
      disabled: IS_AP
    },
```
Nog even disabled voor productie en acceptatie.